views/forms/BarChartFields.jsx

```diff
@@ -94,7 +101,19 @@ const BarChartFields = ({ classes, formValues, setFormValues, studies }) => {
         fullWidth
       />
       <div className={classes.formLabelRow}>
-        <br />
+        <Typography
+          variant='subtitle1'
+          color='textSecondary'
+          gutterBottom={false}
+          className={classes.publicText}
+        >
+          Public
+        </Typography>
+        <Checkbox
+          checked={formValues.public}
+          onChange={updateFormValues}
+          name='public'
+        />
```

Comment:
it seems like the `Typography` is serving as a label to this checkbox, should we use a `label` tag here? alternatively, could we add an `aria-label` or `aria-labelled-by` to the `Checkbox`

Reply:
Thanks for this.
[a9d4afb](https://github.com/AMP-SCZ/dpdash/pull/143/commits/a9d4afbc6bffe85e5131e8547ab31a657102d5a9)